### ISSUE TITLE
sa-vipulā anuṣṭubh

### ISSUE THREAD

**Issue by danbalogh** [erc-dharma/project-documentation#332]
Our list of permitted metres does not include the sa-vipulā variant of the anuṣṭubh. As far as I know, this vipulā is not recognised by any authority on chandas, but it occurs in some texts frequently enough that at least some modern theoreticians believe it to have been a legitimate variant at least in some circles. I have now run into such a line in one of my texts, where a metrical error is more likely (the same stanza has a grossly unmetrical quarter too), but I see that `met="sa-vipulā"` has been used in at least one DHARMA file, DHARMA_INSCIK00667.xml, and we may want to legitimise that.

Should I add the sa-vipulā to the list of prosodic patterns?

**Comment by arlogriffiths** [comment]
Yes, please add it. I have now come across a case that may be sa or ra vipulā depending on whether one follows actual spelling mūni or corrected muni. It is in https://dharmalekha.info/texts/INSCIK01574.

`<l n="a" real="-++---+="><lb n="A1"/>śilā-pañca-m<choice><sic>ū</sic><corr>u</corr></choice>ni-kalpaḥ</l>`

**Comment by danbalogh** [comment]
OK, let's keep this issue open in hopes that I'll find it at some point and do the needful.

**Comment by danbalogh** [comment]
I've added the sa-vipulā and, for good measure, the ta-vipulā, to the authority file.
